packtools/sps/validation/related_articles.py

```diff
@@ -40,13 +46,11 @@ def related_articles_matches_article_type_validation(self, correspondence_list=N
                     'xpath': './article[@article-type] .//related-article[@related-article-type]',
                     'validation_type': 'match',
                     'response': 'OK',
-                    'expected_value': 'A valid match such as the following: (correction-forward, corrected-article) or ('
-                                      'retracted-article, retraction-forward)',
-                    'got_value': 'article-type: correction-forward, related-article-type: corrected-article',
-                    'message': 'Got (correction-forward, corrected-article), expected (correction-forward, '
-                               'corrected-article) or (retracted-article, retraction-forward)',
+                    'expected_value': ['corrected-article'],
+                    'got_value': 'corrected-article',
+                    'message': 'Got corrected-article, expected one of the following items: corrected-article',
```

Comment:
No lugar de 'corrected-article', você coloca a lista para concordar com o texto (one of the following items)

Reply:
OK @robertatakenaka